### ISSUE TITLE
Can't use the trial version

### ISSUE THREAD

**Issue by HuubBuis** [MetalWorkerTools/CNCR#1]
**Describe the bug**
The trial version 1.2.9 does not allow anything to do, all buttons are grey

This is a bug that has been fixed in release 1.2.11 Release 1.2.11 is currently in review by the Microsoft Store.

I expect that the Microsoft Store will release this version within 1 to 3 days.

**Comment by HuubBuis** [comment]
Microsoft claims suddenly to be unable to remove their own app license and install their own trial license. Without a trial license they can't test a trial version and so they won't release a new version that offers a trial license.
I had to remove the trial license (offer) from the store to release the app (version 1.2.11) still having trial license capability. This allows users, that recently installed the trial version, to update the app and use it until their trial license expires.
I have no idea what Microsoft wants to accomplish and certainly can't solve their issue. In a few months I will try again, maybe they behave normal again. Until then, I can't offer a trial license any more.

**Comment by HuubBuis** [comment]
I can't get Microsoft to release a combination of trial version and usage licenses.  Every question to Microsoft results in an advice to ask "**the other**" department. In the end, I gave up.
Having an trial version is more valuable to new users than the choice of a one year or unlimited usage license. So I have decided to drop the usage licenses. Anyone  (current and new users) having an app license, can use the app unlimited.
